gcloud_bigtable/row.py

```diff
@@ -416,12 +419,13 @@ def _commit_mutate(self, timeout_seconds=None):
             mutations=mutations_list,
         )
         timeout_seconds = timeout_seconds or self.timeout_seconds
-        response = self.client.data_stub.MutateRow.async(request_pb,
-                                                         timeout_seconds)
         # We expect a `._generated.empty_pb2.Empty`.
-        response.result()
+        if async:
+            self.client.data_stub.MutateRow.async(request_pb, timeout_seconds).result()
```

Comment:
Sounds about right.

@joerideturck What issues was `async` causing you?

----

**ASIDE**: I am upgrading to beta in the near future, so much of this will be moot.